### ISSUE TITLE
java.lang.NoClassDefFoundError: org/apache/commons/jrcs/rcs/ParseException

### ISSUE THREAD

**Issue by szepeviktor** [oracle/opengrok#808]
Could help me?
tomcat 7 + Oracle JRE 1.7.0_51-b13 + opengrok 0.12-rc4

```
debian-v:/opt/opengrok# bin/OpenGrok index
Loading the default instance configuration ...
Exception: org/apache/commons/jrcs/rcs/ParseException
00:01:17 SEVERE: Unexpected Exception
java.lang.NoClassDefFoundError: org/apache/commons/jrcs/rcs/ParseException
        at org.opensolaris.opengrok.history.RepositoryFactory.<clinit>(RepositoryFactory.java:42)
        at org.opensolaris.opengrok.index.Indexer.main(Indexer.java:456)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.jrcs.rcs.ParseException
        at java.net.URLClassLoader$1.run(Unknown Source)
        at java.net.URLClassLoader$1.run(Unknown Source)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        ... 2 more
```

Source tree is in /var/opengrok/src

**Comment by Ricky54326** [comment]
I just had this same issue - it happened because I copied opengrok.jar  to another folder before running it, ignorantly. After running it in its original path ($ROOT/lib/opengrok.jar), the error went away for me.

**Comment by Dev0Null** [comment]
@Ricky54326 Thanks for the tip. This resolved said issue which is was testing using 0.12.1.5 (stable).
